_data/changelogs/component-header.yml

```
@@ -2,6 +2,13 @@ title: Header
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Allow mobile nav to stay open during page loads to prevent confusion upon closing.
+    summaryAdditional:  Links to subsection headers will still appropriately close the mobile navigation window.
```

Wondering if we can shorten this a bit. 
```suggestion
    summary: Updated the behavior of the mobile header navigation to stay open while loading a new page.
    summaryAdditional:  
```